src/frontends/tensorflow/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenVINO TensorFlow Frontend
+
+The TensorFlow Frontend (TF FE) is a C++ based OpenVINO Frontend component that is responsible for reading and converting TensorFlow models to the Intermediate Representation (IR) format.
+
+```mermaid
+flowchart BT
+    model[(model.pb)]
+    style model fill:#427cb0
+    model2[(model.pb)]
+    style model2 fill:#427cb0
+    ovtf(OpenVINO Integration with TensorFlow)
+    style ovtf fill:#ffffc2
```

Comment:
click ovtf "https://github.com/openvinotoolkit/openvino_tensorflow"

Reply:
I will add it.

Reply:
Done.